MinGL.cpp

```diff
@@ -169,6 +170,103 @@ void MinGL::putPixel(int x, int y, const MinGLColor& color, int width, int heigh
 	glDisable(GL_SCISSOR_TEST);
 }
 
+const MinGLColor MinGL::getPixelColor(int x, int y, int width, int height) const{
+	int pixelSize = 4; // RGBA has 4 components (R, G, B, A)
```

Comment:
Please use constexpr whenever possible, like here

Reply:
used constexpr for `pixelsize`  and static constexpr for `epsilon` in MinGLColor struct